player.md

```
@@ -119,6 +119,50 @@ Starts playing. If the player is not initialized, it will be initialized automat
 
+## Event listeners
+
+The Player proxies selected Sequencer events with a `sequencer.` namespace prefix. Payloads are passed through unchanged.
+
+### `sequencer.item_started_playing`
+
+Emitted when a playlist item has started playing.
+
+**Payload:**
+
+| Property | Type   | Description                    |
```

markdownlint-cli2 (0.20.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: leading_and_trailing; Unexpected leading pipe

(MD055, table-pipe-style)